RELEASE-NOTES.md

```diff
@@ -4,6 +4,61 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+-
+
+### Components:
+- **ContextSelector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
```

Comment:
This structure  does not match previous release notes.  "Updated signature...." should be on a new line and have a bullet.

Reply:
Thank you!